### PR TITLE
Bundle js

### PR DESCRIPTION
**Pull request by svillafe**
## Changes
* Added landingBundle.es6 to the project. This file contains the necessary js for the bahaviour of the landing pages.

### A no-menu

a-no-menu

### A menu

a-menu

### A selection

a-selection

### B no-menu

b1-no-menu

### B menu

b1-menu

### B selection

b1-selection

### CODE REVIEW

File: frontend/assets/javascripts/src/modules/landingBundles.es6 [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+    });
+}
+
+function toggle(section) {
+    if(getComputedStyle(section).display === 'none') {
+        section.style.display = 'block';
+    } else {
+        section.style.display = 'none';
+    }
```
Comment (paulbrown1982) [inline frontend/assets/javascripts/src/modules/landingBundles.es6]: Just to remove the boilerplate on this you could rewrite something like:

section.style.display = (getComputedStyle(section).display === 'none') ? 'block' : 'none';

File: frontend/app/views/fragments/bundle/bundleButton.scala.html [hunk 2]
```diff
@@ -1,9 +1,9 @@
 @import views.support.Asset
 
-@(buttonLabel: String, target: String, dataLink: String, chevron: Boolean = false, secondary: Boolean = false)
+@(buttonLabel: String, target: String, className: Option[String] = None, chevron: Boolean = false, secondary: Boolean = false)
 
 <div class="bundle-button">
-    <a data-link-name="@dataLink" class="@if(secondary){ bundle-button__content--secondary } else { bundle-button__content }" href="@target">
+    <a class="@if(secondary){ bundle-button__content--secondary } else { bundle-button__content } @className.getOrElse("")" href="@target">
```
Comment (paulbrown1982) [inline frontend/app/views/fragments/bundle/bundleButton.scala.html]: Rather than .getOrElse("") you can just use .mkString

**Comment by paulbrown1982** [comment]
👍 from a HTML/JS/CSS perspective

File: frontend/app/views/fragments/bundle/bundleOfferingsA.scala.html [hunk 3]
```diff
@@ -106,59 +107,59 @@
...
                                     <p></p>
                                 </div><div class="bundle-offering-a__print-option__price">
-                                    £23.40/month
+                                    @bundleVariant.prettyMonthlyPrice(GuardianWeekly)
                                 </div>
                             </li>
                             <li class="bundle-offering-a__subscribe__offer__print-options__option">
                                 <div class="bundle-offering-a__print-option__description">
                                     <h1>Weekend</h1>
                                     <p>Saturday & Sunday</p>
```
Comment (rupertbates) [inline frontend/app/views/fragments/bundle/bundleOfferingsA.scala.html]: Use `&amp;`

File: frontend/app/views/fragments/bundle/bundleOfferingsA.scala.html [hunk 4]
```diff
@@ -106,59 +107,59 @@
...
                                 </div><div class="bundle-offering-a__print-option__price">
-                                    £26.00/month
+                                    @bundleVariant.prettyMonthlyPrice(Weekend)
                                 </div>
                             </li>
                             <li class="bundle-offering-a__subscribe__offer__print-options__option">
                                 <div class="bundle-offering-a__print-option__description">
                                     <h1>Saturday & The&nbsp;Guardian Weekly</h1>
```
Comment (rupertbates) [inline frontend/app/views/fragments/bundle/bundleOfferingsA.scala.html]: Use `&amp;`

File: frontend/app/views/fragments/bundle/bundleOfferingsA.scala.html [hunk 5]
```diff
@@ -58,12 +59,12 @@
                             </div>
                         </li>
                     </ul>
-                    @fragments.bundle.bundleButton("Find out more", routes.Bundle.thankYou(bundleVariant, "Digital Subscription").url,"")
+                    @fragments.bundle.bundleButton("Find out more", routes.Bundle.thankYou(bundleVariant, "[A Design] Digital Subscription").url)
                 </div>
             </div><div class="bundle-offering-a__subscribe__offer">
                 <div class="bundle-offering-a__subscribe__offer__content">
                     <h2>Print&Digital subscription</h2>
```
Comment (rupertbates) [inline frontend/app/views/fragments/bundle/bundleOfferingsA.scala.html]: Use `&amp;`

**Comment by svillafe** [comment]
It is working on **PROD**

**Pull request merged**